homework/zad8_result.txt

```diff
@@ -0,0 +1,128 @@
+
+Number of elements in vector: 1000
+Minimum elements for one thread: 200
+Number of hardware threads: 2
+
+pararelAccumulate() result: 500500
+std::accumulate() result: 500500
+
+Create threads: 11065us
+Wait for finish threads: 0us
+
+Pararel algorithm: 11332us
+
+Normal algorithm: 415us
+
+ ----- end test ----
+
+Number of elements in vector: 1000000
+Minimum elements for one thread: 200
+Number of hardware threads: 2
+
+pararelAccumulate() result: 1784293664
+std::accumulate() result: 1784293664
+
+Create threads: 13221us
+Wait for finish threads: 0us
+
+Pararel algorithm: 242826us
+
+Normal algorithm: 356427us
+
+ ----- end test ----
+
+Number of elements in vector: 1000000
+Minimum elements for one thread: 256
+Number of hardware threads: 2
+
+pararelAccumulate() result: 1784293664
+std::accumulate() result: 1784293664
+
+Create threads: 11588us
+Wait for finish threads: 0us
+
+Pararel algorithm: 238095us
+
+Normal algorithm: 348425us
+
+ ----- end test ----
+
+Number of elements in vector: 1000
+Minimum elements for one thread: 500
+Number of hardware threads: 2
+
+pararelAccumulate() result: 500500
+std::accumulate() result: 500500
+
+Create threads: 11220us
+Wait for finish threads: 0us
+
+Pararel algorithm: 11402us
+
+Normal algorithm: 397us
+
+ ----- end test ----
+
+Number of elements in vector: 1000000
+Minimum elements for one thread: 500
+Number of hardware threads: 2
+
+pararelAccumulate() result: 1784293664
+std::accumulate() result: 1784293664
+
+Create threads: 12438us
+Wait for finish threads: 0us
+
+Pararel algorithm: 236967us
+
+Normal algorithm: 337272us
+
+ ----- end test ----
+
+Number of elements in vector: 1000000
+Minimum elements for one thread: 512
+Number of hardware threads: 2
+
+pararelAccumulate() result: 1784293664
+std::accumulate() result: 1784293664
+
+Create threads: 13076us
+Wait for finish threads: 0us
+
+Pararel algorithm: 232354us
+
+Normal algorithm: 344745us
+
+ ----- end test ----
+
+Number of elements in vector: 1000000
+Minimum elements for one thread: 2000
+Number of hardware threads: 2
+
+pararelAccumulate() result: 1784293664
+std::accumulate() result: 1784293664
+
+Create threads: 8705us
+Wait for finish threads: 0us
+
+Pararel algorithm: 223151us
+
+Normal algorithm: 340171us
+
+ ----- end test ----
+
+Number of elements in vector: 1000000
+Minimum elements for one thread: 2048
+Number of hardware threads: 2
+
+pararelAccumulate() result: 1784293664
+std::accumulate() result: 1784293664
+
+Create threads: 11291us
+Wait for finish threads: 0us
```

Comment:
te opisy nie mówią już o tym co te countery faktycznie mierzą.
W szczególności wait for finish ;)

Reply:
OK